app/jobs/perilune/tasks/executor_job.rb

```diff
@@ -72,15 +78,33 @@ def tracer
       end
 
       def track_stats(success:)
-        count_hash = success ? { count: 1, success: 1 } : { count: 1, failure: 1 }
-        inner_hash = { task.task_klass.downcase.intern => count_hash }
+        task_type = task.task_type.downcase == 'import' ? 'import' : 'export'
+        state = success ? 'success' : 'failure'
+
         Trifle::Stats.track(
-          key: "perilune::#{task.task_type.downcase}::#{task.task_klass.downcase}",
+          key: "perilune::#{task_type}::#{state}",
           at: Time.zone.now,
           config: Perilune.default.stats_driver_config,
-          values: count_hash.merge(inner_hash)
+          values: trifle_values_hash(success:, task_type:)
         )
       end
+
+      def duration_to_ms
+        return 0 unless @duration.present? && !@duration.zero?
+
+        (@duration * 1000).round
+      end
```

Comment:
As mentioned before, we convert duration to `ms` and we `round` it. We need to have a global overviews of durations, success and fails. We don't need here to have ultra deep accuracy with 10 decimals.